api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -28,6 +29,7 @@
 @Service
 public class NotebooksServiceImpl implements NotebooksService {
 
+  private static final long MAX_NOTEBOOK_READ_SIZE_BYTES = 5 * 1000 * 1000; // 5MB
```

Comment:
can this be a config value instead?

Reply:
Yes, but I prefer it to not be in this case. My policy on when one should use a flag vs constant:

- Is the value likely to change?
- Is there likely to be a need to change the value dynamically / out of band of a release?

In this case, I'd say no and no, which makes me prefer a const. The disadvantages of using a flag are:
- Introduces an additional moving part which your tests/devs need to cover
- Obfuscates the runtime value for a reader
- Adds additional configuration code

This case is definitely borderline, but I will default to a constant unless given a reason to do otherwise.